storybooks/native/themes/themes.hsl.js

```diff
@@ -27,7 +27,9 @@ const hslTheme = {
   quoteTextColor: colors.primary.hslGreyDark,
   letterSpacing: '-0.5px',
   activatablePointerHeight: '0.6rem',
-  activatableLineHeight: '4px'
+  activatableLineHeight: '4px',
+  error: '#dc0451',
```

Comment:
I think these colors should come from ./color.js. I added misc-section into the feature/colors -branch